integration_test/gce/gce_testing.go

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build integration_test
```

Comment:
probably want to put this back because it's causing unit tests to fail (more init()  stuff)

Reply:
Yep, I'll do that. I wish we could refactor the package to not use the `init()` but it's way too big to do in this PR